src/python/WMCore/WebTools/RESTModel.py

```diff
@@ -185,6 +185,14 @@ def wrapper(*input_args, **input_kwargs):
             if len(args) != 0:
                 # store the method name
                 method = methodKey
+                # for PUT/POST requests data may be send in body of the request
+                # we check if this is the case (no args/kwargs) and read body
+                if  verb in ['PUT', 'POST'] and \
+                    not input_args and not input_kwargs:
+                    from WMCore.Wrappers import JsonWrapper
+                    body = cherrypy.request.body.read()
+                    if  body:
+                        input_kwargs = JsonWrapper.loads( body )
```

Comment:
Hi Valentin, actually this block is causing the problem.  when it is converted input or input_kwargs it is automatically checks the number of argument with the corresponding method. Basically params passed in the body need to be handled each method not in the RESTModel. This will break all the calls using WebTools/RESTModel.py (including DBS calls)